packages/core/admin/ee/server/constants/default-workflow.json

```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "name": "Default"
```

Comment:
Quick question: why is the default color not in here too?

Reply:
workflow does not have a color no? 
For stages, https://github.com/strapi/strapi/blob/feature/review-workflow-name/packages/core/admin/ee/server/constants/default-stages.json I planned not to add the default color but to add different colors to each stage (based on a discussion with Maeva)